stories/components/Pill.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Pill from '../../lib/Pill';
+import Icon from '../../lib/Icon';
+import StoryItem from '../styleguide/StoryItem';
+
+storiesOf('Components', module).add('Pill', () => (
+  <div>
+    <StoryItem title="Pill">
+      <Pill>
```

Comment:
You had a button in this before to show clearing. That is still valid for the stories as you'll need styling to handle component X + Y working together.